src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 16px;
+
+  transition-duration: $transition-duration;
```

Comment:
It would be great to add transition-properties too
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

Reply:
How exactly would you recommend doing that? If I try to write transition: [property-name] $transition-duration; for Each relevant property, I recieve a mistake